releasenotes/notes/1.3/deprecate-basic-simulator-configuration-9d782925196993e9.yaml

```
@@ -8,7 +8,7 @@ deprecations_providers:
     (``backend.target``).
 
     Here is a quick guide for accessing the most common ``BackendConfiguration`` attributes in the 
-    :class:`BackendV2` model:""
+    :class:`BackendV2` model:
```

This should format the table as a code snippet and respect the structure:

```suggestion
    :class:`BackendV2` model::
```